ivy/functional/ivy/device.py

```diff
@@ -767,36 +767,25 @@ def tpu_is_available() -> bool:
 
 # noinspection PyShadowingNames
 @handle_exceptions
-def default_device(
-    device: Optional[Union[ivy.Device, ivy.NativeDevice]] = None,
-    /,
-    *,
-    item: Optional[Union[list, tuple, dict, ivy.Array, ivy.NativeArray]] = None,
-    as_native: bool = None,
-) -> Union[ivy.Device, ivy.NativeDevice]:
+def default_device(device=None, *, item=None, as_native=None):
     """
     Return the input device or the default device. If the as_native flag is set, the
     device will be converted to a native device. If the item is provided, the item's
     device is returned. If the device is not provided, the last default device is
     returned. If a default device has not been set, the first gpu is returned if
-    available, otherwise the cpu is returned.
+    available, otherwise the cpu is returned..
```

Comment:
```suggestion
    available, otherwise the cpu is returned.
```

Reply:
this was a mistake, which I thought I removed